metricflow/validation/data_warehouse_model_validator.py

```diff
@@ -135,10 +136,14 @@ def renderize(
         return (rendered_plan.sql, rendered_plan.bind_parameters)
 
     @classmethod
-    def gen_semantic_model_tasks(cls, manifest: SemanticManifest) -> List[DataWarehouseValidationTask]:
+    def gen_semantic_model_tasks(
+        cls, manifest: SemanticManifest, filter_by_semantic_models: Optional[Sequence[str]] = None
```

Comment:
nit: filter_by_semantic_models sounds like a boolean option, maybe semantic_models_filter is better?